composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "require": {
-        "onpayio/php-sdk": "^1.0",
+        "onpayio/php-sdk": "1.0.2",
```

Comment:
Let's not change this, the sdk is semantic versioned so no need to specify a specific version here.